src/main/resources/templates/admin/dashboard.html

```
@@ -408,135 +855,244 @@ <h2>RAW 상품 관리</h2>
+                                </div>
+                                <p class="text-danger x-small mt-1 mb-0"><i class="fas fa-info-circle me-1"></i>삭제된 데이터는 복구할 수 없습니다.</p>
+                            </form>
+                            <div id="raw-product-delete-result" class="api-result"></div>
+                        </div>
+                    </div>
+                </div>
             </div>
         </div>
     </div>
 </div>
+</div>
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </div> ]

(tag-pair)